lms/djangoapps/verify_student/services.py

```diff
@@ -116,15 +116,17 @@ def get_expiration_datetime(cls, user, statuses):
             'status__in': statuses,
         }
 
-        photo_id_verifications = SoftwareSecurePhotoVerification.objects.filter(**filter_kwargs)
-        sso_id_verifications = SSOVerification.objects.filter(**filter_kwargs)
-        manual_id_verifications = ManualVerification.objects.filter(**filter_kwargs)
+        sort_by = 'updated_at'
+
+        photo_id_verifications = SoftwareSecurePhotoVerification.objects.filter(**filter_kwargs).order_by('-' + sort_by)
```

Comment:
We need to order each queryset first before we try to compare values across the querysets. This is because within `most_recent_verification`, we only grab the first record from each queryset.